core/src/test/java/org/apache/iceberg/TestMetrics.java

```diff
@@ -73,6 +72,7 @@ public static List<Object> parameters() {
   }
 
   @TempDir public Path temp;
```

Comment:
Unrelated nit
```suggestion
  @TempDir private Path temp;
```

Reply:
can't make it private unfortunately, but protected would work, since it's being used in subclasses